.gitignore

```diff
@@ -20,10 +20,13 @@ var/*
 .tox
 .DS_Store
 .coverage
+.coverage.*.*
```

Comment:
Suppresses `git status` complaining of new files if tox is cancelled mid-test.

Reply:
Would `.coverage*` be a better ignore?

Reply:
That will exclude `.coveragerc` as well, though. The default "partial coverage filename format" is, [according to coverage.py documentation](https://coverage.readthedocs.io/en/coverage-5.3/cmd.html#combining-data-files-coverage-combine), `.coverage.${HOSTNAME}.${PID}.${RANDOM}`.

The two periods there I think was back when the config file was still named `.coverage.ini`, so I had to use two periods to not exclude the config file.

If we want to simplify that exclusion, we still need to leave in at least one period so we won't accidentally exclude `.coveragerc`

Reply:
This is merged in #195, so I'm marking as resolved.